opentelemetry-sdk/src/opentelemetry/sdk/metrics/__init__.py

```diff
@@ -116,6 +107,21 @@ def get_handle(self, label_values: Sequence[str]) -> BaseHandle:
         self.handles[label_values] = handle
         return handle
 
+    def update(
+        self, label_values: Sequence[str], value: metrics_api.ValueT
+    ) -> None:
+        """ Generic method used to update the metric value.
+
+        Used primarily for batch operations. Implementations
```

Comment:
```suggestion
        Used primarily for batch operations. Implementations should call the
```

More wrapping nits, I know we need a linter/formatter for this.